### PR TITLE
Barnacle instructions

### PR DESCRIPTION
**Pull request by tanaes**
Added some resources and updated readme for executing on Barnacle.

### CODE REVIEW

File: cluster_configs/barnacle/envs.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+anvi: source activate oecophylla-anvi
+humann2: source activate oecophylla-humann2
+qc: source activate oecophylla-qc
```
Comment (qiyunzhu) [inline cluster_configs/barnacle/envs.yml:3]: qc and raw use the same conda env. Is this intentional?

**Review by qiyunzhu — APPROVED** [review]
I see no problem with this PR.

**Review by mortonjt — APPROVED** [review]
@tanaes is there anything else that is needed before merging?

**Comment by tanaes** [comment]
NOt at the moment, please do merge!

**Pull request merged**